src/Features/Core/Portable/Completion/Providers/AbstractAwaitCompletionProvider.cs

```diff
@@ -21,62 +24,127 @@ namespace Microsoft.CodeAnalysis.Completion.Providers
     /// </summary>
     internal abstract class AbstractAwaitCompletionProvider : LSPCompletionProvider
     {
+        #region CompletionItem properties keys
+        private const string AwaitCompletionTargetTokenPosition = nameof(AwaitCompletionTargetTokenPosition);
+
+        private static class AwaitCompletionChange
+        {
+            public const string AddAwaitAtCursor = nameof(AwaitCompletionChange) + "." + nameof(AddAwaitAtCursor);
+            public const string AddAwaitBeforeDotExpression = nameof(AwaitCompletionChange) + "." + nameof(AddAwaitBeforeDotExpression);
+            public const string AppendConfigureAwait = nameof(AwaitCompletionChange) + "." + nameof(AppendConfigureAwait);
+            public const string MakeContainerAsync = nameof(AwaitCompletionChange) + "." + nameof(MakeContainerAsync);
```

Comment:
why do you need all these to be prefixed with `nameof(AwaitCompletionChange).`?

Reply:
Not really. I just wanted to "namespace" the items in the property bag that belong to this particular CompletionProvider to distinguish them from other common keys like "description", "position"  and the like.